pkgs/applications/office/docear/default.nix

```diff
@@ -0,0 +1,48 @@
+{stdenv, fetchurl, runtimeShell
+, oraclejre
+, antialiasFont ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "docear";
+  version = "1.2";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "http://docear.org/downloads/docear_linux.tar.gz";
+    sha256 = "1g5n7r2x4gas6dl2fbyh7v9yxdcb6bzml8n3ldmpzv1rncgjcdp4";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+  buildInputs = [ oraclejre ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share
+    cp -R * $out/share
+    chmod 0755 $out/share/ -R
+
+    # The wrapper ensures oraclejre is used
+    echo "#!${runtimeShell}" > $out/bin/docear
+    ${stdenv.lib.optionalString antialiasFont ''
+      echo "export _JAVA_OPTIONS='-Dswing.aatext=TRUE -Dawt.useSystemAAFontSettings=on'" >> $out/bin/docear
+    ''}
+    echo "export JAVA_HOME=${oraclejre.home}" >> $out/bin/docear
+
+    echo "cd $out/share/" >> $out/bin/docear
+    echo "${runtimeShell} docear.sh" >> $out/bin/docear
+    chmod 0755 $out/bin/docear
+    '';
+
+  meta = with stdenv.lib; {
+    description = "A unique solution to academic literature management";
+    homepage = http://www.docear.org/;
```

Comment:
Please enclose in quotes.

Reply:
Is this a new practice? Most URLs in nixpkgs are actually unquoted.

Reply:
Yes. Very new. Unquoted urls are going to eventually be deprecated. You don't *have* to quote it right now, but minimal effort now saves some time later when it has become deprecated.